### DISCUSSION TITLE
pull git repository after server start

### DISCUSSION THREAD

**Discussion by moriokii** [GameServerManagers/LinuxGSM#3908] [Q&A]
pull git repository after server start is that possible?

**Comment by Claiyc** [comment]
You don't want to pull the repo in order to setup / maintain a gameserver. Follow the installation instructions on http://linuxgsm.com

I honestly don't see any reason why to pull the repo manually if you don't want to develop lgsm

**Reply by moriokii** [reply]
to make development easier, when server starts, it pulls server files, like gamemode or other

**Reply by Claiyc** [reply]
So you mean updating the server on server start? That is implemented via updateonstart="on" in the lgsm config.

Updating lgsm is handled via the update-lgsm command. That probably shouldn't be done on every server start.

**Reply by moriokii** [reply]
Not updating lgsm, i want to connect repo with my server content, gamemods, addons etc to and pull commits on server start

**Reply by Claiyc** [reply]
Still not sure what you are trying to accomplish. As a general answer: You should not change any files (pull something external) in the serverfiles/ directory if the server is running (after serverstart) and no lgsm files, while a lgsm command is being executed. Does this answer your question?
